Tests/Realm.Tests/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/AsymmetricObjectWithAllTypes_generated.cs

```diff
@@ -704,8 +706,8 @@ public string RequiredStringProperty
                 }
             }
 
-            private string _stringProperty;
-            public string StringProperty
+            private string? _stringProperty = null!;
```

Comment:
Why are we generating ` = null!` for strings that are optional?

Reply:
Fixed